source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -84,6 +84,9 @@ To launch the OZWCP web application:
   Don't check the USB box regardless of using a USB based device.
 </p>
 
+### {% linkable_title Using the GPIOs %}
```

Comment:
Needs a empty line after to have the formating function properly.
With that exception looks good. 
The AiO really needs some love to have this added by default.